packages/template-retail-react-app/config/default.js

```diff
@@ -36,7 +36,9 @@ module.exports = {
             // This differs from the siteId in commerceAPIConfig for testing purposes
             siteId: 'aaij-MobileFirst',
             isProduction: false
-        }
+        },
+        // if this is true, useOrigin will take x-forwarded-host header into consideration
+        enableXForwardedHost: false
```

Comment:
From our recent discussion, we plan to remove this config, is that right?

Reply:
good catch. It is removed